### ISSUE TITLE
✨ Add a new column to the Requests table

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#6603]
## ✨ Feature

DCM relied on the group and level used in the  pool title to track their request, adding the group, level and stream in one column from the request details into a column will help DCM better track their requests.

In addition we can create better use of space by merging the link of "View request" with the "Job title" 

## 🕵️ Details

Add a new column that includes the following details from the request summary:
- Group and Level + Selected stream

Merge the Action column and the Job title column, this will make the job title the clickable link to view the individual request.

## 🎨 Design File

[Include a link to the design file.](https://www.figma.com/file/Bc5ZqiNbQfNrhAqJHGT09o/Request-page?type=design&node-id=101%3A9122&t=6WUM1XHbxDZMxnML-1)

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/732559f0-58a0-4fea-859a-5c669421a83e)

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Criteria 1
- [ ] Criteria 2

**Comment by tristan-orourke** [comment]
@Jerryescandon just wondering, why do you want Stream in the same column as Group+level? Why not two columns?

**Comment by tristan-orourke** [comment]
@Jerryescandon is this okay for the ordering if you sort by "Group, level and stream"?
> Sorting by this column orders rows primarily by group (alphabetically), then by level within those groups, and then by stream (ordered by enum name, so basically English name)

**Comment by gobyrne** [comment]
Search bar is primarily used to find requests from a specific "manager" by recruitment team.

**Comment by gobyrne** [comment]
Need to filter by Status, Stream and Level.

**Comment by gobyrne** [comment]
@Jerryescandon after discussion with Recruitment team we decided to break Stream into it's own column. Could you please update figma and screenshot?

**Comment by Jerryescandon** [comment]
Yeah, relying on filters for the new columns is probably the best bet.

**Comment by tristan-orourke** [comment]
#6955 requests further columns added - it can be combined with this one
